Remotion/Core/Core/Utilities/NameValueCollectionUtility.cs

```diff
@@ -24,7 +24,7 @@ namespace Remotion.Utilities
   /// </summary>
   public static class NameValueCollectionUtility
   {
-    public static NameValueCollection Clone (this NameValueCollection collection)
+    public static NameValueCollection Clone (this NameValueCollection? collection)

```

Comment:
agreed. This was rewritten because the compiler is not smart enough to know that first cant be null after
```csharp
      if (first == null && second == null)
        return null;
      else if (first != null && second == null)
        return Clone (first);
      else if (first == null && second != null)
        return Clone (second);
```